### DISCUSSION TITLE
Can I integrate Plausible Analytics CE into my product?

### DISCUSSION THREAD

**Discussion by Nikola-Milovic** [plausible/analytics#4906] [Self-Hosted Support] [answered]
With the new CE and its AGPLv3 licence, what I can and cannot do with Plausible Analytics? I wanted to integrate into my product for basic analytic purposes for my clients, basically count the page views and then get them via api. 

I understand that AGPLv3 licence basically means that I should contribute back any code changes that I make, but I don't plan on actually modifying anything. I plan on using the CE as a standalone service that provides API that I interact with, I am not sure how this aligns with AGPLv3.

Thanks in advance! If not I'll look for another solution or create a simple one by myself to track page views, but I'd like to avoid this if possible

**Comment by ruslandoga** [comment]
👋 

I think it should be OK. I'll double check with the core team ~~on Wednesday.~~ The core team has replied already :)

**Comment by ukutaht** [comment, marked as answer]
Yes the way you plan to use it works fine with our license 👍 Good luck!
